containers/kraken.yml

```diff
@@ -18,7 +18,7 @@ spec:
         - name: kraken
           securityContext:
             privileged: true
-          image: quay.io/openshift-scale/kraken
+          image: bmos299/kraken-debug
```

Comment:
We might want to skip updating the image, an image build will be triggered once we merge this PR: https://quay.io/repository/openshift-scale/kraken.

Reply:
absolutely...i didnt mean to commit this.